tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -85,7 +85,7 @@ protected TracingQueryGateway(Builder builder) {
     @Override
     public <R, Q> CompletableFuture<R> query(String queryName, Q query, ResponseType<R> responseType) {
         Span parentSpan = tracer.activeSpan();
-        try (Scope scope = tracer.buildSpan(queryName).startActive(false)) {
+        try (Scope scope = tracer.buildSpan("send"+SpanUtils.messageName(query,queryName)).startActive(false)) {
```

Comment:
super nit: couple of missing space (format code)

Reply:
formatted code